examples/jet/pages/index.tsx

```diff
@@ -98,29 +98,19 @@ function AuthedHome() {
 
   return (
     <>
-      <Head>
-        <link rel="preconnect" href="https://fonts.googleapis.com" />
-        <link
-          rel="preconnect"
-          href="https://fonts.gstatic.com"
-          crossOrigin="true"
-        />
-        <link
-          href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
-          rel="stylesheet"
-        />
-      </Head>
       <div className={`flex flex-col h-screen bg-${theme}`}>
-        <div className="flex flex-row justify-end p-2 items-center space-x-2">
-          <NotificationCenterButton
-            wallet={wallet}
-            network={'devnet'}
-            publicKey={DIALECT_PUBLIC_KEY}
-            theme={theme}
-            variables={themeVariables}
-          />
-          <WalletButton />
-        </div>
+        <DialectProvider
+          wallet={wallet}
+          network={'devnet'}
+          publicKey={DIALECT_PUBLIC_KEY}
```

Comment:
arbitrary naming issue but if this is a jet example it should be called `JET_PUBLIC_KEY`